README.md

```diff
@@ -3,11 +3,13 @@
 
 # Chasqui
 
-Chasqui is a simple, lightweight, persistent implementation of the publish-subscribe (pub-sub)
-messaging pattern for service oriented architectures.
+Chasqui is a simple, lightweight, persistent implementation of the
+publish-subscribe (pub-sub) messaging pattern for service oriented
+architectures.
```

Comment:
@kellyredding I really appreciate your feedback. I haven't pushed it yet, but there's a big documentation refresh coming (including documenting all the public methods and configuration options). The README is still very much in a just get it out there state. One of the things I want to change is go from zero to working in the README with as little fluff as possible. Now that you've pointed it out, I agree that "simple, lightweight" are just marketing fluff and don't contribute to that goal.